modules/ROOT/nav.adoc

```diff
@@ -21,7 +21,7 @@
 *** xref:static-ip-config.adoc[Configuring a Static IP Address]
 *** xref:sysctl.adoc[Kernel Tuning]
 *** xref:running-containers.adoc[Running Containers]
-*** xref:authentication.adoc[Configuring Authentication]
+*** xref:configuring-users.adoc[Configuring Users and Groups]
```

Comment:
The rename will break internal xrefs (which we can fix) as well as external links (which we can't, of course).  In general I think we should try to avoid changing page URLs.

Reply:
Good point; I'll leave the filename as is and just rename the section.